src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -2020,6 +2035,13 @@ int wm_vuldet_report_agent_vulnerabilities(sqlite3 *db, agent_software *agent, w
         goto end;
     }
 
+    int sock = wm_vuldet_get_wdb_socket();
+
+    //Clean the vulnerabilities in the agent database
+    if (OS_INVALID == wdb_agents_vuln_cve_clear(atoi(agent->agent_id), &sock)) {
+        mtwarn(WM_VULNDETECTOR_LOGTAG, "Failed to remove vulnerabilities from the agent %s database", agent->agent_id);
```

Comment:
Maybe a word like _clear_ is more related to the real purpose of the function than remove

Reply:
Fixed